src/include/daos_cont.h

```diff
@@ -712,11 +723,24 @@ daos_cont_destroy_cpp(daos_handle_t poh, const uuid_t cont, int force, daos_even
 	uuid_unparse(cont, str);
 	return daos_cont_destroy2(poh, str, force, ev);
 }
+
+#define daos_cont_create daos_cont_create_cpp
+static inline int
+daos_cont_create_cpp(daos_handle_t poh, uuid_t *uuid, daos_prop_t *cont_prop, daos_event_t *ev)
+{
```

Comment:
Please add a case for daos_cont_create to src/client/api/compile_check.c just to ensure it works for c++ now and into the future.